data/json/npcs/Kindred/NPC_Darren_Cooper.json

```
@@ -0,0 +1,613 @@
+    "traits": [
+      { "trait": "OUTDOORSMAN" },
+      { "trait": "INSOMNIA" },
+      { "trait": "GOODCARDIO2" },
+      { "trait": "STRONGSTOMACH" },
+      { "trait": "SPIRITUAL" },
+      { "trait": "TERRIFYING" },
+      { "trait": "KILLER" },
+      { "trait": "seer_mark" },
+      { "group": "Appearance_Caucasian" },
+      { "trait": "hair_brown_short" },
+      { "trait": "FACIAL_HAIR_3DAYSTUBBLE" }
+    ],
+    "common": false,
+    "bonus_per": { "rng": [ 0, 2 ] },
+    "bonus_str": { "rng": [ 0, 2 ] },
+    "worn_override": "KINDRED_COOPER_worn",
+    "weapon_override": "KINDRED_COOPER_wield",
+    "shopkeeper_item_group": "NC_KINDRED_COOPER_misc",
+    "skills": [
+      {
+        "skill": "ALL",
+        "level": { "mul": [ { "one_in": 3 }, { "sum": [ { "dice": [ 2, 2 ] }, { "constant": -2 }, { "one_in": 4 } ] } ] }
+      },
+      { "skill": "dodge", "bonus": { "rng": [ 2, 4 ] } },
+      { "skill": "melee", "bonus": { "rng": [ 5, 7 ] } },
+      { "skill": "cutting", "bonus": { "rng": [ 4, 6 ] } }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "KINDRED_COOPER_worn",
+    "subtype": "collection",
+    "entries": [
+      { "item": "loincloth_leather" },
+      { "item": "socks" },
+      { "item": "kevlar" },
+      { "item": "helmet_liner" },
+      { "item": "gloves_liner" },
+      { "item": "boots_larmor" },
+      { "item": "swat_armor" },
+      { "item": "gauntlets_larmor" },
+      { "item": "leather_belt" },
+      { "item": "canteen" },
+      { "item": "armguard_larmor" },
+      { "item": "backpack_leather" },
+      { "item": "knee_pads" },
+      { "item": "knit_scarf" }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "KINDRED_COOPER_wield",
+    "subtype": "collection",
+    "entries": [ { "item": "survivor_machete" } ]
+  },
+  {
+    "type": "item_group",
+    "id": "NC_KINDRED_COOPER_misc",
+    "subtype": "distribution",
+    "entries": [
+      { "item": "needle_bone", "prob": 40 },
+      { "item": "bone_glue", "prob": 20 },
+      { "item": "bone_flute", "prob": 10 },
+      { "item": "bone_human", "prob": 10 },
+      { "item": "broth_bone", "prob": 60 },
+      { "item": "water_clean", "prob": 90 }
+    ]
+  },
+  {
+    "type": "npc",
+    "id": "KINDRED_Darren_Cooper",
+    "//": "A once member of a doomsday cult.",
+    "name_unique": "Darren Cooper",
+    "gender": "male",
+    "name_suffix": "Rogue Kindred",
+    "class": "NC_KINDRED_COOPER",
+    "attitude": 0,
+    "mission": 7,
+    "chat": "TALK_KINDRED_COOPER",
+    "faction": "no_faction"
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_KINDRED_COOPER",
+    "dynamic_line": {
+      "u_has_var": "talked_to_cooper",
+      "type": "dialogue",
+      "context": "first_meeting",
+      "value": "yes",
+      "no": {
+        "u_has_trait": "seer_mark",
+        "yes": "How do you do.  Name's Cooper.  What's your- wait…  that's the Mark of the Seer on your hand.  You're Kindred.  So I guess you know who I am then.",
```

```suggestion
        "yes": "How do you do.  Name's Cooper.  What's your- wait…  that's the Mark of the Seer on your hand.  You're Kindred.  Maybe you've heard of me then.",
```
Up to you, but it seems to me Darren wouldn't necessarily be certain everyone knows him on sight despite a connection to the kindred.

---

I see what you mean, but the way I feel, "I guess" does leave it more as an assumption, while he also feels like things have gone so far with Brigitte that she would tell all her Kindred to stop him if possible. "maybe you've heard of me" to me feels more like he's a celebrity and wants people to know him, whereas this is more him afraid/sad that you do.